addons/website/static/src/js/utils.js

```diff
@@ -416,6 +416,24 @@ function generateGMapLink(dataset) {
         + '&ie=UTF8&iwloc=&output=embed';
 }
 
+/**
+ * Checks if the current device is mobile.
+ *
+ * @private
+ * @param {Object} self - context object ("this")
+ * @returns {boolean}
+ */
+function isMobile(self) {
+    let isMobile;
+    self.trigger_up('service_context_get', {
+        callback: (ctx) => {
+            isMobile = ctx['isMobile'];
+        },
+    });
+
+    return isMobile;
```

Comment:
Actually maybe `isMobile ? true : <call web_editor utils>` ? We don't normally fully support editing the website from a mobile device... but I guess we will be going in the direction that we could at some point... would justify a bit more having an util for this. Either do that or remove this util entirely (and do what it does in the _isMobile override directly).